### PR TITLE
Add new parser for /etc/nfs.conf

### PR DESCRIPTION
**Pull request by JoySnow**
Signed-off-by: XiaoXue Wang <xiaoxwan@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

Add new spec and parser for file /etc/nfs.conf .

### CODE REVIEW

**Comment by JoySnow** [comment]
Hi @xiangce , could you help review this MR?

**Comment by JoySnow** [comment]
Hi @xiangce , the "Note" is added for "no combiner". Could you help review the MR?

File: insights/parsers/nfs_conf.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+"""
+
+from insights.core.plugins import parser
+from insights.core import IniConfigFile
+from insights.specs import Specs
+
+
+@parser(Specs.nfs_conf)
+class NFSCONF(IniConfigFile):
```
Comment (bfahr) [inline insights/parsers/nfs_conf.py:20]: @JoySnow the name should be `NFSConf` with mixed case shouldn't it?

**Review by bfahr — CHANGES_REQUESTED** [review]

File: insights/parsers/nfs_conf.py (hunk 1 above)
Comment (JoySnow) [reply insights/parsers/nfs_conf.py]: Renamed to "NFSConf".

**Review by xiangce — APPROVED** [review]

**Pull request merged**